src/winkerberos.c

```diff
@@ -475,7 +609,15 @@ sspi_client_step(PyObject* self, PyObject* args) {
         return NULL;
     }
 
-    result = auth_sspi_client_step(state, challenge);
+    if (pychan_bindings != NULL) {
+        if (!PyCObject_Check(pychan_bindings)) {
+            PyErr_SetString(PyExc_TypeError, "Expected a SecPkgContext_Bindings object");
+            return NULL;
+        }
+        sec_pkg_context_bindings = (SecPkgContext_Bindings *)PyCObject_AsVoidPtr(pychan_bindings);
```

Comment:
Do we have to check for NULL, if `pychan_bindings` isn't set through the kwargs it will send NULl anyway to `auth_sspi_client_step`. If I do check for NULL would I be checking `pychan_bindings` or the result of this pointer cast?